stages/org.osbuild.dracut

```diff
@@ -83,12 +83,8 @@ def main(tree, options):
         if initoverlayfs:
             initfs_bin = "/usr/bin/initoverlayfs-install"
 
-        with ChrootProcDevSys(tree):
-            subprocess.run(["/usr/sbin/chroot", tree, initfs_bin,
```

Comment:
linter is now complaining that "W0611: Unused import subprocess (unused-import)"

Reply:
Woopsie!